docs/dependencies.md

```diff
@@ -48,3 +48,30 @@ and checks if the value is bigger or equal than 5 and that result `is_valid` is
 {! ../docs_src/_shared/exceptions.md !}
 
 The same is applied also to [exception handlers](./exception-handlers.md).
+
+## More Real World example
+
+Now let's imagine that we have some web application with one of the `views.py` as:
+
+```python hl_lines="16"
+{!> ../docs_src/dependencies/views.py !}
+```
+
+As you can notice the `user_dao` is injected automatically using appropriate level of dependency injection.
+
+Let's see the `urls.py` and understand from where we got the `user_dao`:
+
+```python hl_lines="14-16 32-34"
+{!> ../docs_src/dependencies/urls.py !}
+```
+
+Here we use lambdas to create a callable from DAO instance.
+> Read more about DAOs here: [Protocols](./protocols.md)
```

Comment:
I would put Read more about [DAOs](./protocols.md) instead of the here. Just straight I would say. The same applied to the rest 🙂